src/cpu/aarch64/acl_pooling.cpp

```diff
@@ -15,39 +15,284 @@
 *******************************************************************************/
 
 #include "cpu/aarch64/acl_pooling.hpp"
+#include "common/memory_tracking.hpp"
 
 namespace dnnl {
 namespace impl {
 namespace cpu {
 namespace aarch64 {
 
+status_t acl_pooling_fwd_t::pd_t::init(engine_t *engine) {
+
+    // Compute Library supports forward propagation only
+    bool ok = set_default_params() == status::success && is_fwd()
+            && utils::everyone_is(src_md()->data_type, dst_md()->data_type)
+            && utils::one_of(
+                    src_md()->data_type, data_type::f32, data_type::f16)
+            && attr()->has_default_values()
+            && attr_.set_default_formats(dst_md(0)) == status::success
+            && !is_dilated() && !has_zero_dim_memory();
+
+    ACL_CHECK_SUPPORT(!ok, "ACL supports forward propagation only");
```

Comment:
This error message will be confusing since `ok` might be false even for forward propagation (e.g. if `dt == s8`). Just print something more generic like "unsupported primitive options".